### PR TITLE
Initialize project with serverless framework

### LINKED ISSUE #1: Serverless Boilerplate

### LINKED ISSUE #3: Serverless config
Config based on aws free tier

### PR DESCRIPTION
**Pull request by RealDyllon**
Resolves #1 
Resolves #3 

Linked to #2

### CODE REVIEW

**Comment by RealDyllon** [comment]
@tayalaks2001 Anything missing from this PR?

**Comment by tayalaks2001** [comment]
> @tayalaks2001 Anything missing from this PR?

Nope, all good!

**Comment by cyiafn** [comment]
Reviewer can approve merge also esp if tests pass

**Comment by RealDyllon** [comment]
@tayalaks2001 mind updating the readme?

**Comment by tayalaks2001** [comment]
@RealDyllon updated the docs, I think this PR is ok to be merged now!

**Comment by RealDyllon** [comment]
@yankai14 could you review this PR? cheers

File: be/api/v1/templates/nonAuthRoute.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from fastapi import FastAPI
+from mangum import Mangum
+import os
+
+
+def createNonAuthRouter(router):
+    stage = os.environ.get('STAGE', 'dev')  # ???
```
Comment (yankai14) [inline be/api/v1/templates/nonAuthRoute.py:7]: Remove the comment?

**Review by yankai14 — CHANGES_REQUESTED** [review]

File: be/api/v1/templates/nonAuthRoute.py (hunk 1 above)
Comment (RealDyllon) [reply be/api/v1/templates/nonAuthRoute.py]: @yankai14 
Fixed :)

**Review by yankai14 — APPROVED** [review]

**Pull request merged**